### PR TITLE
Remove size from xml

### PR DESCRIPTION
**Pull request by brianteeman**
Pull Request for Issue #35711  .

> size
Valid for email, password, tel, url and text input types only. Specifies how much of the input is shown. Basically creates same result as setting CSS width property with a few specialities. The actual unit of the value depends on the input type. For password and text, it is a number of characters (or em units) with a default value of 20, and for others, it is pixels. **CSS width takes precedence over size attribute.**

As css takes precedence over any size attribute it has been suggested that we remove all size="xx" from the xml form definitions.

Personally I would keep it for all frontend inputs as there may not be any css in the site template for this but I would remove all the admin input sizes as they are redundant.

@chmst @bembelimen

### CODE REVIEW

**Comment by chmst** [comment]
> Personally I would keep it for all frontend inputs as there may not be any css in the site template for this but I would remove all the admin input sizes as they are redundant.

Not sure if I understand correctly. 
You want to let all fields in components and modules unchanged and change only fields in backend? 
Yes, agree.

**Comment by brianteeman** [comment]
This PR removes the size option from all fields that are used in the admin because css takes precedence

This PR does **not** remove the size option from fields displayed in the frontend because there may not be css.

(at least that is what it is supposed to do)

**Comment by jwaisner** [comment]
@brianteeman , do we just need to test everything for this one or are there specific areas / test instructions?

**Comment by brianteeman** [comment]
Really all you need to do is to open any admin form and check that there has been no change in the display.

File: components/com_contact/tmpl/contact/default.xml [hunk 1]
```diff
@@ -38,7 +38,7 @@
 			<field
 				name="show_contact_category"
 				type="list"
-				label="COM_CONTACT_FIELD_CONTACT_SHOW_CATEGORY_LABEL"
+				label="COM_CONTACT_FIELD_CONTACT_SHOW_CATEGORY_LABEL1"
```
Comment (Quy) [inline components/com_contact/tmpl/contact/default.xml]:
```suggestion
				label="COM_CONTACT_FIELD_CONTACT_SHOW_CATEGORY_LABEL"
```

Comment (brianteeman) [reply components/com_contact/tmpl/contact/default.xml]: oops

**Comment by Quy** [comment]
Please resolve conflicts.

**Comment by brianteeman** [comment]
conflicts resolved

File: administrator/components/com_menus/forms/item.xml [hunk 2]
```diff
@@ -116,8 +106,7 @@
 			name="menuordering"
 			type="MenuOrdering"
 			label="COM_MENUS_ITEM_FIELD_ORDERING_LABEL"
-			filter="int"
-			size="1">
+			filter="int">
 		</field>
```
Comment (Quy) [inline administrator/components/com_menus/forms/item.xml]:
```suggestion
			filter="int"
			>
		</field>
```

Comment (brianteeman) [reply administrator/components/com_menus/forms/item.xml]: actually both are wrong

Comment (Quy) [reply administrator/components/com_menus/forms/item.xml]: It is a dropdown (select), thus I assume it is valid `>`.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 7ea38a839ed4819e9b04932c47e9ee97dfe91a42<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35742">issues.joomla.org/tracker/joomla-cms/35742</a>.</sub>

**Comment by tecpromotion** [comment]
I have tested this item :white_check_mark: successfully on 7ea38a839ed4819e9b04932c47e9ee97dfe91a42<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35742">issues.joomla.org/tracker/joomla-cms/35742</a>.</sub>

**Comment by tecpromotion** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35742">issues.joomla.org/tracker/joomla-cms/35742</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35742">issues.joomla.org/tracker/joomla-cms/35742</a>.</sub>

**Comment by brianteeman** [comment]
rebased to 4.1

**Comment by bembelimen** [comment]
Thx

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**